readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://artuom2283>.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://artuom2283.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="957" alt="image" src="https://github.com/user-attachments/assets/cb6dd272-d1a8-4f55-934d-31cee5a16085">
```suggestion
- [DEMO LINK](https://artuom2283.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://artuom2283.github.io/layout_search-bar-airbnb/report/html_report/)
```